### PR TITLE
SYNCOPE-1661 added sidebar layout JSON configuration

### PR DESCRIPTION
**Pull request by andrea-patricelli**

### CODE REVIEW

**Comment by ilgrosso** [comment]
@andrea-patricelli please add the issue number to this PR title, e.g. rename it as `[SYNCOPE-XXX] ,`

File: client/idrepo/enduser/src/main/java/org/apache/syncope/client/enduser/pages/SelfPasswordReset.java [hunk 1]
```diff
@@ -91,10 +91,7 @@ public SelfPasswordReset(final PageParameters parameters) {
             @Override
             protected void onSubmit(final AjaxRequestTarget target) {
                 boolean checked = true;
```
Comment (ilgrosso) [inline client/idrepo/enduser/src/main/java/org/apache/syncope/client/enduser/pages/SelfPasswordReset.java:93]: @andrea-patricelli it seems that `checked` can be safely removed

Comment (andrea-patricelli) [reply client/idrepo/enduser/src/main/java/org/apache/syncope/client/enduser/pages/SelfPasswordReset.java:93]: You're right. Going to remove

**Comment by ilgrosso** [comment]
@andrea-patricelli a test is failing:

 AuthenticatedITCase.selfUpdate:99 » NullPointer

**Review by ilgrosso — APPROVED** [review]

**Pull request merged**